scripts/us_ca_chhs/healthcare_facility_bed_counts/preprocess_csv_helper.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import numpy as np
+
+# This function is to return a map{county_name: geoId} given CA county name.
```

Comment:
```suggestion
# Return a map{county_name: geoId} given a California county name.
```

We like to use imperative sentences since the subject is implicitly the function.  Q.v., https://en.wikipedia.org/wiki/Imperative_mood.